src/cider/exec/nextgen/context/CodegenContext.h

```diff
@@ -54,6 +82,10 @@ class CodegenContext {
     return arrow_array_values_.size();
   }
 
+  jitlib::JITValuePointer getContentPtr(int64_t id,
```

Comment:
```id``` should only be used in CodegenContext internally and not be exposed to the caller, move this one to private field. BTW, I think ```getBufferContentPtr``` is more clear.